nemo_text_processing/text_normalization/taggers/electronic.py

```diff
@@ -34,8 +34,19 @@ class ElectronicFst(GraphFst):
     def __init__(self):
         super().__init__(name="electronic", kind="classify")
 
-        username = pynutil.insert("username: \"") + NEMO_SIGMA + pynutil.insert("\"") + pynini.cross('@', ' ')
-        domain_graph = pynutil.insert("domain: \"") + NEMO_SIGMA + pynutil.insert("\"")
+        username = (
+            pynutil.insert("username: \"")
+            + NEMO_ALPHA
+            + pynini.closure(NEMO_ALPHA | NEMO_DIGIT | NEMO_NOT_SPACE)
```

Comment:
could you also use symbols.tsv in here for username and domain ? to prevent the input being in NEMO_NOT_SPACE\symbols

do we need all the symbols in tsv? e.g. can "{" appear in email?

Reply:
Based on wiki, `{` could appear username, but domain can only take `.` or `-`